src/cargo/ops/registry.rs

```diff
@@ -205,7 +205,7 @@ pub fn registry(config: &Config,
         src.update().chain_err(|| {
             format!("failed to update {}", sid)
         })?;
-        (src.config()?).unwrap().api
+        (src.config()?).unwrap().api.unwrap()
```

Comment:
Could this avoid the `unwrap` and return an error instead?

Reply:
Its guaranteed not to fail (unless you've manually corrupted your index or something) because the src we're accessing is crates.io.

Reply:
Ah true yeah, but this may help future-proof for custom registries?